conf/application-json-logger.xml

```diff
@@ -9,7 +9,7 @@
 
```

Comment:
Whilst we are looking at logging, can we change our ones to be configurable? 
See https://github.com/hmrc/tai-frontend/blob/master/conf/application-json-logger.xml for an example 
and how we can adjust it in the base config here:

https://github.com/hmrc/app-config-base/blob/6400a1c550a7f06fe67bd95948f503a488832627/tai-frontend.conf#L32

and if you ever wonder how uk.gov loggers ignore DEBUG, its set to INFO in the common-frontend config here: 
https://github.com/hmrc/app-config-common/blob/d1c3a3d6285fd144aa93427e6ef2ab4ec85b25f0/build-frontend-common.yaml#L83 

Here is another example of us overriding the logger levels in the production config: 
https://github.com/hmrc/app-config-production/blob/63bac4569ea477c87d7c0270a614770296ebdb9b/pertax-frontend.yaml#L53